docs/code/use_dataset_in_paddlepaddle.py

```diff
@@ -5,6 +5,7 @@
 
 # pylint: disable=pointless-string-statement
 # pylint: disable=wrong-import-position
+# pylint: disable=duplicate-code
```

Comment:
these disable comments should be put into another PR

Reply:
Yes, just because the PR behind may not report these errors. If we find the solution, it will be put into another PR.